### PR TITLE
Add Rainbow & coinbase wallet adapters

### PR DESCRIPTION
**Pull request by chaitanyapotti**
Adds coinbase & rainbow wallet adapters

### CODE REVIEW

**Comment by himanshuchawla009** [comment]
hey @tainguyentt can u test it on android phone, i will test on IOS.

File: packages/adapters/wallet-connect-v1-adapter/src/walletConnectV1adapter.ts [hunk 1]
```diff
@@ -192,25 +193,32 @@ class WalletConnectV1Adapter extends BaseAdapter<void> {
...
+    if (chainId !== parseInt(this.chainConfig.chainId, 16)) {
+      try {
+        await this.wcProvider.switchChain({ chainId: this.chainConfig.chainId, lookup: false });
+      } catch (error) {
+        log.error(error);
+        // we need to create a new session since old session is already used and
+        // user needs to login again with correct chain with new qr code.
+        await this.createNewSession({ forceNewSession: true });
+        const connectedChainConfig = getChainConfig(CHAIN_NAMESPACES.EIP155, chainId);
+        this.emit(
```
Comment (himanshuchawla009) [inline packages/adapters/wallet-connect-v1-adapter/src/walletConnectV1adapter.ts:211]: any reason why we are not throwing this error along with emitting it.

Comment (chaitanyapotti) [reply packages/adapters/wallet-connect-v1-adapter/src/walletConnectV1adapter.ts:211]: Didn't change the previous logic

**Review by himanshuchawla009 — APPROVED** [review]
lgtm

**Pull request merged**